subcommand/server-acl-init/command.go

```diff
@@ -233,7 +233,7 @@ func (c *Command) getConsulServers(logger hclog.Logger) *apiv1.PodList {
 				return fmt.Errorf("no server pods with labels %q found", labelSelector)
 			}
 
-			if len(serverPods.Items) < c.flagReplicas {
+			if len(serverPods.Items) < n {
 				return fmt.Errorf("found %d servers, require %d", len(serverPods.Items), c.flagReplicas)
```

Comment:
Should this error message use `n` instead of `c.flagReplicas` as well?

Reply:
🍺